atomic_reactor/plugins/post_compare_components.py

```diff
@@ -5,13 +5,24 @@
 This software may be modified and distributed under the terms
 of the BSD license. See the LICENSE file for details.
 """
+import logging
```

Comment:
Unnecessary since you're calling the log object in self.

Reply:
I used logging.WARNING in method as optional paramter